src/test/java/de/rabea/UI/UserInterfaceTest.java

```diff
@@ -40,12 +40,11 @@ public void displaysEmptyBoard() {
 
     @Test
     public void displaysBoardWithMarks() {
-        Board board = new Board();
-        board.placeMark(1, Cell.X);
-        board.placeMark(3, Cell.O);
+        board.placeMark(1, Mark.X);
+        board.placeMark(3, Mark.O);
         userInterface.displayBoard(board.cells());
         assertThat(fakeConsole.messagePrinted()).isEqualTo(
-                "\n" +
+                "\033[H\033[2J\n" +
```

Comment:
maybe extracting `"\033[H\033[2J\n"` into a method clearScreenSymbols() would help explain what the random letters/numbers are to someone who doesn't know.